Projekty/Projekt1/Kacper_Grzymkowski_Jan_Gaska/Inzynieria Cech, wstępne modelowanie/WUM_Kacper_Grzymkowski_Jan_Gaska_Kamien_Milowy_II.ipynb

```diff
@@ -0,0 +1,2102 @@
+{
```

Comment:
<p>Z uwagi na to, że zbiór jest niezbalansowany warto byłoby się zastanowić nad użyciem parametru stratify w ramach train_test_split</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>